views/partials/offering/detail-relationship.jade

```diff
@@ -0,0 +1,25 @@
+.row
+  .col-xs-12
+    .container-flex
```

Comment:
This is pretty similar to the HTML used in `views/partials/offering/search-grid.jade`. Please, DRY!